styles/components/_layout.scss

```diff
@@ -90,6 +90,30 @@
   flex-direction: column;
 }
 
+.pureFlex {
+  display: flex;
```

Comment:
What are the advantages of the pureFlex class over the existing classes?

There are two things we want to achieve with the CSS to consider here:

1. we want to avoid needing a long list of classes on each element. Classes should have more than one property in most cases, and if more than 3 non-color classes are needed on an element, a single class should be created that incorporates those properties. 

2. we want to enforce visual consistency, which we can do by using consistent classes. We should be very hesitant to add more layout classes if classes already exist that do the same thing. Whenever possible the minimum number of classes should be used, as that will make the UX consistent, vs. lots of slightly different classes which will make the UX inconsistent.